test/lib/mayaUsd/nodes/testProxyShapeBase.py

```diff
@@ -23,7 +23,7 @@
 import os
 import unittest
 
-import usdUtils, mayaUtils
+import usdUtils, mayaUtils, ufeUtils
```

Comment:
Maya 2019 and 2020 don't have ufeSupport plugin, so we need to use a fallback.

https://github.com/Autodesk/maya-usd/blob/0b7a931b8149afacfafd1027cb655158ff400ebe/test/testUtils/ufeUtils.py#L28